### PR TITLE
Adds pre-commit prettier

### PR DESCRIPTION
**Pull request by Pingviinituutti**
Added it to root so that the pre-commit git hook runs prettier on staged changes. Also adds the `yarn format` command in the root, if you want to run prettier on whole repo.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "trailingComma": "none",
```
Comment (jhnj) [inline .prettierrc:2]: Just because you can doesn't mean you should IMO :)

**Comment by jhnj** [comment]
Discussion with @thesauri : pre-commit hook should not be mandatory. Instead we'll add a github action dry running prettier. Is there a way to make the husky stuff opt-in? Otherwise I'd vote for a combination of our PRs: `yarn format` from here and the simple `pre-commit` hook from my PR

**Comment by Pingviinituutti** [comment]
> > Discussion with @thesauri : pre-commit hook should not be mandatory. Instead we'll add a github action dry running prettier. Is there a way to make the husky stuff opt-in? Otherwise I'd vote for a combination of our PRs: `yarn format` from here and the simple `pre-commit` hook from my PR
> 
> `yarn install --production` doesn't install the dev dependencies... :D

Husky suggests using Lerna for repos with multiple projects https://github.com/typicode/husky#monorepos

Lerna: https://github.com/lerna/lerna

**Comment by jhnj** [comment]
Yeah, I've read about Lerna. However, not sure if it's worth the hassle in this case, maybe. If I'd get to choose we'd use bazel :)

I think mine and @thesauri 's opinion is that we don't want commit hooks by default and I think we'd still want `yarn format` in the root so `install --production` is not an option.

**Comment by jhnj** [comment]
@Pingviinituutti discussed with @thesauri and we'll just add a github action to check the formatting instead. Closing this and #37

**Pull request closed without merging**